DQM/BeamMonitor/plugins/FakeBeamMonitor.cc

```diff
@@ -891,26 +889,6 @@ void FakeBeamMonitor::FitAndFill(const LuminosityBlock& lumiSeg, int& lastlumi,
 
   std::vector<float> DipPVInfo_;
   DipPVInfo_.clear();
-  //
-  //  if (countTotPV_ != 0) {
-  //    DipPVInfo_.push_back((float)countEvtLastNLS_);
-  //    DipPVInfo_.push_back(h_nVtx_st->getMean());
-  //    DipPVInfo_.push_back(h_nVtx_st->getMeanError());
-  //    DipPVInfo_.push_back(h_nVtx_st->getRMS());
-  //    DipPVInfo_.push_back(h_nVtx_st->getRMSError());
-  //    DipPVInfo_.push_back((float)MaxPVs);
-  //    DipPVInfo_.push_back((float)countTotPV_);
-  //    MaxPVs = 0;
-  //  } else {
-  //    for (size_t i = 0; i < 7; i++) {
-  //      if (i > 0) {
-  //        DipPVInfo_.push_back(0.);
-  //      } else {
-  //        DipPVInfo_.push_back((float)countEvtLastNLS_);
-  //      }
-  //    }
-  //  }
-  //  theBeamFitter->SetPVInfo(DipPVInfo_);
```

Comment:
@francescobrivio, please confirm it's fine to remove this commented section.

Reply:
Thanks Marco for reminding me!
Yes it's fine to remove these commented lines from the BeamSpot point of view!

Reply:
Thanks @mmusich, @francescobrivio